lte/protos/mconfig/mconfigs.proto

```diff
@@ -372,6 +372,8 @@ message SubscriberDB {
 
     // Enable relaying S6a messages via FeG RPC
     bool hss_relay_enabled = 6;
+
```

Comment:
Add a description of the field, including what it represents (secs?)

Reply:
Added. Thanks!